src/sections/Community/Handbook/about.js

```diff
@@ -168,10 +222,22 @@ const Intro = () => {
                 </span>
               </li>
             </ol>
-
           </div>
+          <TocPagination />
         </Container>
-        <TocPagination />
+        <div>
+          <div className="intra-page">
+            <ul>
+              {content.map((x) => (
```

Comment:
I tried , when i import the component i am unable to style it to the location , u can try , i tried a lot but it work, it always was in the place where the toc is , even after doing everything it din't move . @toth2000

Reply:
Wait i missed something , i will try to do it again.

Reply:
@toth2000 Check the code now, it would be good , the only mistake i was making was i made the component const instead of a function 😂😂, thank you for the review.

Reply:
@YashKamboj sometimes it happens.